src/settingsdialogplugins.cpp

```diff
@@ -87,9 +122,18 @@ void PluginsSettingsTab::filterPluginList()
       auto* item = topLevelItem->child(j);
       auto* plugin = this->plugin(item);
 
-      if (m_filter.matches([plugin](const QRegularExpression& regex) {
+      // Check the item or the child - If any match (item or child), the whole
+      // group is displayed.
+      bool match = m_filter.matches([plugin](const QRegularExpression& regex) {
         return regex.match(plugin->localizedName()).hasMatch();
-        })) {
+      });
+      for (auto* child : m_pluginContainer->requirements(plugin).children()) {
```

Comment:
This loop doesn't do anything, the return value from `matches()` is ignored. I'm also not clear as to what it's supposed to be doing.

Reply:
This is suppose to filter from children plugin. E.g. if you search for "Patches", this should match "FNIS Patches Tool", but since "FNIS Patches Tool" is under "FNIS Integration Tool" (with the correct FNIS Tool PR), it won't be find by the outer loop. Should be `match = match || m_filter.matches(...)`.